central/group/datastore/internal/store/store_impl.go

```diff
@@ -298,20 +299,19 @@ func getPossibleGroupProperties(authProviderID string, attributes map[string][]s
 // store for the given auth provider ID, checking whether a default group already exists.
 // If the properties do not indicate a default group or the default group does not yet exist, it will return false.
 // Otherwise, it will return true.
-func checkDefaultGroupForProps(tx *bolt.Tx, props *storage.GroupProperties) (bool, error) {
+func checkDefaultGroupForProps(tx *bolt.Tx, props *storage.GroupProperties) (*storage.Group, error) {
```

Comment:
Please update the function name (`getDefaultGroupForProps()`?) and the comment above.